docset/winserver2022-ps/startlayout/Export-StartLayout.md

```diff
@@ -130,7 +130,10 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 You can specify the following file format: 
 
 - XML.
-Export as an .xml file.
+Export as an .xml file (for Windows 10).
```

Comment:
```suggestion
Export as a .xml file (for Windows 10).
```

Reply:
How do you read out ".xml"? Are you pronouncing the "."? If not, it should be "an" as "x" has a vowel sound.

Reply:
@GenerAhl, yes in this case it would be read as "Export as a **dot** xml file, as opposed to **an** XML file, **an** XML element, etc.